frontend/src/utils.js

```diff
@@ -9,3 +9,7 @@ export function getSponsorMembers(billsResult) {
     coSponsors: coSponsorMembers,
   };
 }
+
+export function createQueryStr(params) {
```

Comment:
IMO this should not be a helper fn. Just inline `new URLSearchParams({ offset, term: searchTerm }).toString()` in `index.jsx`